pandas/tests/test_frame.py

```diff
@@ -13545,6 +13545,18 @@ def test_round(self):
         # Make sure this doesn't break existing Series.round
         tm.assert_series_equal(df['col1'].round(1), expected_rounded['col1'])
 
+        # named columns
+        # GH 11986
+        decimals = 2
+        expected_rounded = DataFrame(
+            {'col1': [1.12, 2.12, 3.12], 'col2': [1.23, 2.23, 3.23]})
+        df.columns.name = "cols"
+        expected_rounded.columns.name = "cols"
+        tm.assert_frame_equal(df.round(decimals), expected_rounded)
+
+        # interaction of named columns & series
```

Comment:
I think that would end up duplicating the existing `Series.round` tests since this change only affects the name of the index representing the columns, not the column names themselves.